src/components/MyRequestCard.jsx

```diff
@@ -15,13 +15,12 @@ const MyRequestCard = ({ request, page }) => {
     if (myActiveRequest) {
```

Comment:
Heads up: you haven't pushed your latest changes I see

Reply:
aaa, thx!